architecture/routing.adoc

```diff
@@ -53,26 +73,40 @@ Routes can be either secure or unsecure. Secure routes provide the ability to us
     "serviceName": "hello-nginx",
     "tls": {
         "termination": "edge",
-        "certificate": "CERT TEXT OMITTED FOR READABILITY",
-        "key": "CERT TEXT OMITTED FOR READABILITY",
-        "caCertificate": "CERT TEXT OMITTED FOR READABILITY"
+        "certificate": "<cert_text>",
+        "key": "<cert_text>",
+        "caCertificate": "<cert_text>"
      }
 }
 ----
-
 ====
 
-Note that while both secure and unsecure is possible with an OpenShift deployment, Red Hat recommends a secure routing service.
+IMPORTANT: While both secure and unsecure routes are possible with an OpenShift
+instance, Red Hat recommends a secure routing service.
 
 == Securing Routes
-Creating a secure route to your pods is accomplished by specifying the TLS Termination of the route and, optionally, providing certificates. At the time of writing, OpenShift Beta 1 TLS termination relies on SNI for serving custom certificates. Any non-SNI traffic received on port 443 has TLS termination with a generic certificate. In the future, the ability to create custom frontends within the router will allow all traffic to serve custom certificates.
+Creating a secure route to your pods is accomplished by specifying the TLS
+termination of the route and, optionally, providing certificates.
```

Comment:
I know this is how it was before you made some edits here, but I think this can be changed to "Create a secure route to your pods by specifying the TLS termination of the route and, optionally, provide any certificates". Or something like this.